File: 0828-chalkboard-xor-game/0828-chalkboard-xor-game.cpp
class Solution {
public:
    bool xorGame(vector<int>& nums) {
        int all_xor = 0;
        int n = nums.size();
        for(int i=0;i<n;i++){
            all_xor ^= nums[i];
        }
        if(all_xor == 0){
            return true;
        }
        if(n%2==0){
            return true;
        }
        return false;

    }
};

File: 0828-chalkboard-xor-game/0828-chalkboard-xor-game.py
class Solution(object):
    def xorGame(self, nums):
        all_xor = 0
        n = len(nums)
        for num in nums:
            all_xor ^= num
        if all_xor == 0:
            return True
        if n % 2 == 0:
            return True
        return False